blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubScm.java

```diff
@@ -70,7 +66,7 @@
     private static final String USER_EMAIL_SCOPE = "user:email";
     private static final String USER_SCOPE = "user";
     private static final String REPO_SCOPE = "repo";
-    private static final String DOMAIN_NAME="github-domain";
+    private static final String DOMAIN_NAME="blueocean-github-domain";
```

Comment:
So, you will need to have a fall-back strategy to cover the case where the admin has disabled the standard per-user store and replaced it with an alternative implementation (think Hashicorp Vault, CyberArk, etc). The alternative implementation is not required to support multiple domains and may indeed decide to only expose all stored credentials in the `global` domain.

Your code should therefore be able to fall back and handle the situation where the add domain fails and either attempt to add the credentials into the global domain or declare that you cannot proceed.